### PR TITLE
Add another entry in iOS 14 changes to point out font rendering issues

### PR DESCRIPTION
**Pull request by xster**
Just remembered that we had this issue https://github.com/flutter/flutter/issues/60013 and users should rebuild with Flutter 1.22 for correct font rendering.

### CODE REVIEW

File: src/docs/development/ios-14.md [hunk 1]
```diff
@@ -26,15 +26,25 @@ but not debug.
...
-ensure that clipboard access notifications are not spuriously 
+ensure that clipboard access notifications are not spuriously
 shown when building text fields.
- 
+
+## System font rendering
+
+If your iOS 14 app uses system fonts such as San Francisco (used by default
+by Cupertino widgets), text will not render correctly and will incorrectly
+be shown in a condensed letter spacing due to changes in iOS's font loading
```
Comment (jmagman) [inline src/docs/development/ios-14.md:36]: maybe "text will incorrectly be rendered in a condensed letter spacing due to..."

File: src/docs/development/ios-14.md [hunk 2]
```diff
@@ -26,15 +26,25 @@ but not debug.
...
+mechanism. This affects production apps on iOS 14.
+
+To ensure correct font renderer, you should build your production apps with
```
Comment (jmagman) [inline src/docs/development/ios-14.md:39]: "font rendering"?

File: src/docs/development/ios-14.md (hunk 1 above)
Comment (xster) [reply src/docs/development/ios-14.md]: 👍

File: src/docs/development/ios-14.md (hunk 2 above)
Comment (xster) [reply src/docs/development/ios-14.md]: 👍

**Review by jmagman — APPROVED** [review]
LGTM

**Comment by xster** [comment]
@sfshaza2 I'm going to merge this since we need to reference this in the blog release. Happy to address any changes after as needed.

**Pull request merged**